flink-python/pyflink/fn_execution/beam/beam_coder_impl_slow.py

```diff
@@ -18,9 +18,11 @@
 
 import datetime
 import decimal
+from types import GeneratorType
```

Comment:
change to `from collections import Iterable, Generator` and put this import after the `import struct`